README.md

```diff
@@ -119,6 +119,19 @@ exports[`CalcComponent should snap 1`] = `
 
 Problems may arise if you're using custom builds (this preset is tailored for `angular-cli` as firsty priority). Please be adivsed that every entry in default configuration may be overriden to best suite your app's needs.
 
+Angular might have problem with Jest regarding to JSDOM missing transform property with Angular material, there is a work around for it. Add this to your `jestGlobalMocks` file
```

Comment:
Can we add a title like `The animation trigger "transformMenu" has failed ` or so like in other cases?
typos: 
* "Angular material" -> "Angular Material"
* "work around" -> "workaround"

Reply:
done :)